ExampleMod/Content/Items/Weapons/ExampleSpear.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ExampleMod.Content.Projectiles;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Content.Items.Weapons
+{
+	public class ExampleSpear : ModItem
+	{
+		public override void SetStaticDefaults() {
+			Tooltip.SetDefault("An example spear");
+		}
+
+		public override void SetDefaults() { 
+			item.damage = 40; // The damage inflicted by the item.
+			item.useStyle = ItemUseStyleID.Shoot; // The use style of your item.
+			item.useAnimation = 18; // The time span of the using animation for the item. 
```

Comment:
similar to above, perhaps `Animation duration, (in frames)` would be more useful (says units). A better comment would say that use style shoot draws the item sprite in the direction of the cursor, for the number of frames specified here.